docs/content/ingestion/misc-tasks.md

```diff
@@ -4,68 +4,6 @@ layout: doc_page
 
 # Miscellaneous Tasks
 
-## Version Converter Task
```

Comment:
The "IndexSpec and Bitmap Types" section belong to "Indexing Service Convert Segment Task" and should also be removed

Reply:
Thanks. Removed.